src/tests/Missions.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { useSelector } from 'react-redux';
+import Missions from '../pages/Missions/Missions';
+
+jest.mock('react-redux');
+
+jest.mock('axios', () => ({
+  get: jest.fn(),
+}));
+
+test('Rockets component renders correctly', () => {
+  useSelector.mockReturnValue({
+    missions: [
+      {
+        id: 1,
+        mission_name: 'Thaicom',
+        description: 'A powerful rocket developed by SpaceX.',
+        reserved: false,
+      },
+    ],
+  });
```

Comment:
- [ ] Good work on adding test cases for the missions. Based on our previous project experience, it's important to wrap your test cases with a meaningful `describe` block. I recommend following our previously gained experience and organizing your test cases within meaningful `describe` blocks for better readability and maintainability.
- [ ] Additionally, you've added test cases for missions, but the descriptions of your test cases mention "rockets," which doesn't accurately reflect what these test cases are testing. To improve clarity, I suggest updating the test case descriptions to provide a more accurate and meaningful description of what each test case is verifying.
- [ ] Furthermore, it's worth noting that out of the 5 tests executed, 1 test failed while 4 tests passed. I recommend investigating the failed test case to identify the issue and update your code to ensure that all test cases pass. Thoroughly debugging and fixing the failed test case will help ensure the reliability of your code.
![image](https://github.com/pushkar1995/space-travelers-hub/assets/55840999/b05eaeb5-8211-4452-b2fd-6d859dd4dd3e)

Overall, your effort to include test cases is commendable, but improving the organization of your tests with `describe` blocks and providing clear test case descriptions will enhance the overall quality of your testing suite. Additionally, addressing the failed test case is crucial to ensure the correctness of your code. Keep up the good work!